CHANGELOG.md

```diff
@@ -1,5 +1,21 @@
 # Changelog
 
+## 35.4.0 [#992](https://github.com/openfisca/openfisca-core/pull/992)
+
+#### New features
```

Comment:
> I'm a little openfisca rusty, but are these really "new features"? Looks more like internal tech debt removal without consequences for users?

Reply:
You're right, there are no technically new features.